src/java/org/apache/cassandra/cql3/transactions/ConditionStatement.java

```diff
@@ -100,12 +100,18 @@ public ConditionStatement prepare(String keyspace, VariableSpecifications bindVa
             
             if (lhs instanceof RowDataReference.Raw)
             {
+                if (((RowDataReference.Raw) lhs).column() == null) {
```

Comment:
2 style comments (to help for future patches)

1) `{` belongs on a new line, this is true almost always (outside of lambda)
2) when the block is a simple line, we can drop the `{}`

Reply:
done. Thanks